mono/metadata/class.c

```diff
@@ -7822,6 +7821,38 @@ mono_class_from_name_checked (MonoImage *image, const char* name_space, const ch
 	return class;
 }
 
+MonoClass *
+mono_class_from_name_checked (MonoImage *image, const char* name_space, const char *name, MonoError *error, MonoGHashTable *visited_images)
+{
+	MonoError e;
+	MonoClass *klass = NULL;
+	gboolean owns_hash;
+
+	if (!error)
```

Comment:
This is wrong.

The MonoError should be the last argument and must never be null.